components/treeview/index.css

```diff
@@ -154,3 +157,8 @@ governing permissions and limitations under the License.
 .spectrum-TreeView-item--indent10 {
   padding-inline-start: calc(10 * var(--spectrum-treeview-item-indent));
 }
+
+.spectrum-TreeView-heading {
+  padding: calc(calc(var(--spectrum-treeview-item-margin-y) * 2) + var(--spectrum-treeview-item-padding-y)) 10px;
+  font-weight: bold;
```

Comment:
We should be getting this font style from DNA.

Reply:
The hardcoed 10px is to align the section text with the Chevron icon.